src/Sylius/Bundle/ShopBundle/Resources/views/Checkout/_steps.html.twig

```diff
@@ -8,7 +8,12 @@
     {% set steps = {'address': 'completed', 'select_shipping': 'completed', 'select_payment': 'completed', 'complete': 'active'} %}
 {% endif %}
 
-<div class="ui four steps">
+{% set isOrderTotalZero = false %}
+{% if orderTotal == 0 %}
+    {% set isOrderTotalZero = true %}
```

Comment:
Do we really need this variable? If so, can't we do something like `{% set isOrderTotalZero = (orderTotal == 0) %}`?